recipe/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "torcheval" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: a498dec34137bc66c9cf1adc7353a46c604dd62255884c72dcb4e2e4fc2cd7e9
+    folder: pypi
+  - git_url: git@github.com:pytorch/torcheval.git
+    git_rev: f1bc22fc67ec2c77ee519aa4af8079f4fdaa41bb
+    git_depth: 1
+    folder: gitproj
+
+build:
+  skip: True  # [py<38]
+  number: 0
+  script:
+    - copy gitproj\*requirements.txt pypi  # [win]
+    - cp gitproj/*requirements.txt pypi    # [not win]
+    - cd pypi
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - git  # [not win]
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - pytorch >=1.11
+
+test:
+  imports:
+    - torcheval
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pytorch.org/torcheval/stable
+  summary: |
+    A library for providing a simple interface to create new metrics and an easy-to-use toolkit for metric
+    computations and checkpointing.
+  description: |
+    A library with simple and straightforward tooling for model evaluations and a delightful user experience. At a high
+    level TorchEval:
+      * Contains a rich collection of high performance metric calculations out of the box. We utilize vectorization and
+        GPU acceleration where possible via PyTorch.
+      * Integrates seamlessly with distributed training and tools using torch.distributed
+      * Is designed with extensibility in mind: you have the freedom to easily create your own metrics and leverage our toolkit.
+      * Provides tools for profiling memory and compute requirements for PyTorch based models.
+  license: BSD-3-Clause
+  license_file: gitproj/LICENSE
```

Comment:
```suggestion
  license_file: LICENSE
```

Reply:
I decided to keep the directory so the two sources are guaranteed to not overwrite each other.